test/test_config.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+import pytest
```

Comment:
Oh, that does look nicer. If unittest weren't baked into the standard library, I would have no issue with replacing it for another test runner. That said, since it also has better hooks into codecov, sounds like a reasonable change.